### ISSUE TITLE
ERROR: memory overlap! [0x812019c0 - 0x812019e0]

### ISSUE THREAD

**Issue by GoogleCodeExporter** [andersonx007/cfg-loader-mod#129]
```
What steps will reproduce the problem?

1.  Use a 3 TB external desktop hard-drive with CFG USB Loader.

2.  Launch any Wii game from CFG USB Loader, v70r65.

3.  Notice memory overlap messages.

What is the expected output? What do you see instead?

The game is expected to launch without issue, but instead I see the following 
error message three times before launch:

Mario Kart Wii

[RMCE01] 2.58GB IOS36

[+] Booting game, please wait...

 IOS(249) .....

 IOS Reload: Blocked

Loading ....ERROR: memory overlap!

dest: 0x812019c0 - 0x812019e0

used: 0x80a8000 - 0x813a7a80

The game then runs without further noted issue.

What version of the product are you using? v70r65

Syscheck, iohash, and debug log attached.

The issue is noted by another user with a 3TB drive and I replied with a more 
in-depth analysis of the issue:

http://gbatemp.net/threads/plz-help-on-using-3tb-external-hdd-on-wii-and-run-met
roid-other-m.334539/#post-4821966

In short, I believe CFG is trying to use a 32-byte memory address (0x812019c0 - 
0x812019e0) that is standardly unavailable with 3 TB hard-drives.
```

Original issue reported on code.google.com by `SpewH...@aol.com` on 6 Nov 2013 at 3:47

Attachments:
- [sysCheck.csv](https://storage.googleapis.com/google-code-attachments/cfg-loader-mod/issue-129/comment-0/sysCheck.csv)
- [ioshash.txt](https://storage.googleapis.com/google-code-attachments/cfg-loader-mod/issue-129/comment-0/ioshash.txt)
- [debug.log](https://storage.googleapis.com/google-code-attachments/cfg-loader-mod/issue-129/comment-0/debug.log)

**Comment by GoogleCodeExporter** [comment]
```
It appears that games located towards the inner edge of the hard-drive platter 
are not effected by this issue.

Instead, only games towards the outer edge of the hard-drive platter appear to 
present the issue.
```

Original comment by `SpewH...@aol.com` on 13 Dec 2013 at 11:02
